GPCS4/Common/BitHelper.h

```diff
@@ -29,7 +29,9 @@ namespace bit {
     return n;
   }
 
+#ifndef _tzcnt_u32
```

Comment:
Why use `#ifdef`? It seems `_tzcnt_u32` is a normal function declaration, not macro.

Reply:
it already defined in .h file, and if not add #ifdef clang does error:
GPCS4\GPCS4\Common\BitHelper.h(33,34): error : C++ requires a type specifier for all declarations
(don't know, how to fix it other way)

Reply:
Can you find `#define _tzcnt_u32` in clang 10 enverienment? I can only find normal declaration in clang 9 and 8.

Reply:
no, it's not #defined, usual `extern` function

Reply:
OK, it seems this is a compatibility bug of clang-cl, search "clang _tzcnt_u32" on google will give many issue reports. It's ok if this approach can fix the problem. https://bugs.llvm.org/show_bug.cgi?id=25544